Dockerfile

```diff
@@ -15,6 +15,9 @@ RUN apt-get update && \
 
 ADD php.ini /usr/local/etc/php/php.ini
 
+# Add required php.ini directives
+RUN echo date.timezone = 'CST6CDT' >> /usr/local/etc/php/php.ini
```

Comment:
To avoid adding another layer to the image, please add this as an additional line to the existing RUN command on line 3:

```
     usermod -u 1000 www-data && \
     echo date.timezone = 'CST6CDT' >> /usr/local/etc/php/php.ini
```